CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Unreleased
```

Comment:
Is it okay for this header to be here in a release commit?

Reply:
That's what [the release guide told me to do](https://github.com/fastly/Viceroy/blob/main/doc/RELEASING.md). :shrug:

Reply:
Ohh the release has already happened, this is approval for the merge. Gotcha!